src/channel.ts

```diff
@@ -472,6 +480,23 @@ const ALL_MARKS: Record<Mark, 'always'> = {
   tick: 'always'
 };
 
+const LABEL_SUPPORTING_MARKS = {
+  circle: 'always',
+  point: 'always',
+  square: 'always',
+  area: 'always',
+  bar: 'always',
+  line: 'always',
+  trail: 'always',
+  rect: 'always'
+} as const;
+
+export type LabelSupportingMark = keyof typeof LABEL_SUPPORTING_MARKS;
+
+export function supportLabel(mark: Mark): mark is LabelSupportingMark {
```

Comment:
Do we need to declare this new method? 

Can we use `getSupportedMark` since it seems to be the same thing?